lib/reporters/html.js

```diff
@@ -260,9 +260,9 @@ function HTML(runner, options) {
 function makeUrl(s) {
   var search = window.location.search;
 
-  // Remove previous grep query parameter if present
+  // Remove previous {grep, fgrep, invert} query parameters if present
   if (search) {
-    search = search.replace(/[?&]grep=[^&\s]*/g, '').replace(/^&/, '?');
+    search = search.replace(/[?&](?:f?grep|invert)=[^&\s]*/g, '').replace(/^&/, '?')
```

Comment:
[Style] Nit: everything else uses `;`, unrelated change ~

```suggestion
    search = search.replace(/[?&](?:f?grep|invert)=[^&\s]*/g, '').replace(/^&/, '?');
```